src/InfoController.php

```diff
@@ -47,6 +51,6 @@ class InfoController
      */
     public function index(DataResponseFactoryInterface $responseFactory): ResponseInterface
     {
-        return $responseFactory->createResponse(['version' => '3.0', 'author' => 'yiisoft']);
+        return $responseFactory->createResponse(['version' => $this->versionProvider->version, 'author' => 'yiisoft']);
```

Comment:
Did not fully understand the necessity of it. Also why not extract author as well then?

Reply:
That's just showcase how to mock dependencies